### DISCUSSION TITLE
force playback, remove "media exceeds device capabilities"

### DISCUSSION THREAD

**Discussion by madmadman** [FossifyOrg/General-Discussion#385] [Q&A]
Is there anyway to remove this "media exceeds device capabilities" behavior? High resolution videos (beyond 1080p) won't generate thumbnails and won't play in the default gallery player stating "media exceeds device capabilities." Meanwhile, I can open and play these videos in VLC just fine. Is there anyway to force the default player to play these videos? Even if it's laggy in the default player, I still want it generate thumbnails and try to play them.

**Comment by naveensingh** [comment]
The error message could be more specific but no, it's not possible to "force playback" in the current implementation.

**Reply by madmadman** [reply]
why? if you don't mind me asking.

**Reply by naveensingh** [reply]
Fossify Gallery uses ExoPlayer under the hood so it can only play videos that are playable using ExoPlayer. Usually, ExoPlayer is able to play 4K/UHD videos if they are _really_ supported by the device.

You may share your device info and if possible, the video file for so we can confirm this isn't an issue with Fossify Gallery itself.

See also:
- https://github.com/FossifyOrg/Gallery/issues/121
- https://github.com/google/ExoPlayer/issues/11218
